src/extensions/extension-discovery.ts

```diff
@@ -393,7 +393,14 @@ export class ExtensionDiscovery extends Singleton {
 
     for (const extension of userExtensions) {
       if ((await fse.pathExists(extension.manifestPath)) === false) {
-        await this.installPackage(extension.absolutePath);
+        try {
+          await this.installPackage(extension.absolutePath);
+        } catch (error) {
+          const message = error.message || error?.toString() || "unknown error";
+          const { name, version } = extension.manifest;
+
+          dialog.showErrorBox("Lens Extension Install Failure", `Failed to install user extension ${name}@${version}: ${message}`);
```

Comment:
Just wondering if dialog is a bit too much, should we just log this error?

Reply:
So with the rest of this PRs change, the extension appears (in the extension page) as active (even though we cannot load it). 

I would like to make all this much more visible to the users but I don't think this PR is the place. I will change this if you would like.